cmake/Modules/SourceFiles.cmake

```diff
@@ -0,0 +1,156 @@
+    ${CMAKE_SOURCE_DIR}/src/childinfo.c
+    ${CMAKE_SOURCE_DIR}/src/defrag.c
+    ${CMAKE_SOURCE_DIR}/src/siphash.c
+    ${CMAKE_SOURCE_DIR}/src/rax.c
+    ${CMAKE_SOURCE_DIR}/src/t_stream.c
+    ${CMAKE_SOURCE_DIR}/src/listpack.c
+    ${CMAKE_SOURCE_DIR}/src/localtime.c
+    ${CMAKE_SOURCE_DIR}/src/lolwut.c
+    ${CMAKE_SOURCE_DIR}/src/lolwut5.c
+    ${CMAKE_SOURCE_DIR}/src/lolwut6.c
+    ${CMAKE_SOURCE_DIR}/src/acl.c
+    ${CMAKE_SOURCE_DIR}/src/tracking.c
+    ${CMAKE_SOURCE_DIR}/src/socket.c
+    ${CMAKE_SOURCE_DIR}/src/tls.c
+    ${CMAKE_SOURCE_DIR}/src/sha256.c
+    ${CMAKE_SOURCE_DIR}/src/timeout.c
+    ${CMAKE_SOURCE_DIR}/src/setcpuaffinity.c
+    ${CMAKE_SOURCE_DIR}/src/monotonic.c
+    ${CMAKE_SOURCE_DIR}/src/mt19937-64.c
+    ${CMAKE_SOURCE_DIR}/src/resp_parser.c
+    ${CMAKE_SOURCE_DIR}/src/call_reply.c
+    ${CMAKE_SOURCE_DIR}/src/script_lua.c
+    ${CMAKE_SOURCE_DIR}/src/script.c
+    ${CMAKE_SOURCE_DIR}/src/functions.c
+    ${CMAKE_SOURCE_DIR}/src/function_lua.c
+    ${CMAKE_SOURCE_DIR}/src/commands.c
+    ${CMAKE_SOURCE_DIR}/src/strl.c
+    ${CMAKE_SOURCE_DIR}/src/connection.c
+    ${CMAKE_SOURCE_DIR}/src/unix.c
+    ${CMAKE_SOURCE_DIR}/src/server.c
+    ${CMAKE_SOURCE_DIR}/src/logreqres.c)
+
+# valkey-cli
+set(VALKEY_CLI_SRCS
+    ${CMAKE_SOURCE_DIR}/src/anet.c
+    ${CMAKE_SOURCE_DIR}/src/adlist.c
+    ${CMAKE_SOURCE_DIR}/src/dict.c
+    ${CMAKE_SOURCE_DIR}/src/valkey-cli.c
+    ${CMAKE_SOURCE_DIR}/src/zmalloc.c
+    ${CMAKE_SOURCE_DIR}/src/release.c
+    ${CMAKE_SOURCE_DIR}/src/ae.c
+    ${CMAKE_SOURCE_DIR}/src/serverassert.c
+    ${CMAKE_SOURCE_DIR}/src/crcspeed.c
+    ${CMAKE_SOURCE_DIR}/src/crccombine.c
+    ${CMAKE_SOURCE_DIR}/src/crc64.c
+    ${CMAKE_SOURCE_DIR}/src/siphash.c
+    ${CMAKE_SOURCE_DIR}/src/crc16.c
+    ${CMAKE_SOURCE_DIR}/src/monotonic.c
+    ${CMAKE_SOURCE_DIR}/src/cli_common.c
+    ${CMAKE_SOURCE_DIR}/src/mt19937-64.c
+    ${CMAKE_SOURCE_DIR}/src/strl.c
+    ${CMAKE_SOURCE_DIR}/src/cli_commands.c)
+
+# valkey-benchmark
+set(VALKEY_BENCHMARK_SRCS
+    ${CMAKE_SOURCE_DIR}/src/ae.c
+    ${CMAKE_SOURCE_DIR}/src/anet.c
+    ${CMAKE_SOURCE_DIR}/src/valkey-benchmark.c
+    ${CMAKE_SOURCE_DIR}/src/adlist.c
+    ${CMAKE_SOURCE_DIR}/src/dict.c
+    ${CMAKE_SOURCE_DIR}/src/zmalloc.c
+    ${CMAKE_SOURCE_DIR}/src/serverassert.c
+    ${CMAKE_SOURCE_DIR}/src/release.c
+    ${CMAKE_SOURCE_DIR}/src/crcspeed.c
+    ${CMAKE_SOURCE_DIR}/src/crccombine.c
+    ${CMAKE_SOURCE_DIR}/src/crc64.c
+    ${CMAKE_SOURCE_DIR}/src/siphash.c
+    ${CMAKE_SOURCE_DIR}/src/crc16.c
+    ${CMAKE_SOURCE_DIR}/src/monotonic.c
+    ${CMAKE_SOURCE_DIR}/src/cli_common.c
+    ${CMAKE_SOURCE_DIR}/src/mt19937-64.c
+    ${CMAKE_SOURCE_DIR}/src/strl.c)
+
+# valkey-rdma module
+set(VALKEY_RDMA_MODULE_SRCS ${CMAKE_SOURCE_DIR}/src/rdma.c)
+
+# valkey-tls module
+set(VALKEY_TLS_MODULE_SRCS ${CMAKE_SOURCE_DIR}/src/tls.c)
+
+# rdma-test source files
+set(VALKEY_RDMA_MODULE_TEST_SRCS ${CMAKE_SOURCE_DIR}/tests/rdma/rdma-test.c)
```

Comment:
What about moving this into `tests/CmakeLists.txt` and `tests/rdma/CmakeLists.txt`(test WITH_RDMA)?

Reply:
> What about moving this into `tests/CmakeLists.txt` and `tests/rdma/CmakeLists.txt`(test WITH_RDMA)?

I can see the logic in your suggestion and also in keeping it here under "SourceFiles" file. I have no strong opinion about it... I will move it